### PR TITLE
service migration and tests for new opentsdb setting

### PR DESCRIPTION
**Pull request by ecifreo**

### CODE REVIEW

File: Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+sm.require("1.0.0")
+log = logging.getLogger("zen.migrate")
+
+
+class UpdateOpenTSDBSkipUnresolvedTags(Migrate.Step):
+    """ Enable OpenTSDB option to skip unresolved tags.
+
+        This migration step iterates through services that have opentsdb.conf as an original
+        configuration file.  For each of these services both original and latest config fles
```
Comment (mozart27) [inline Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py]: nit - `fles` should prolly be `files`

File: Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        """ This method is called by the migration process and overrides the method
+            located in the parent class Migrate.Step.
+        """
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        tsdbs = filter(lambda s: "opentsdb" in ctx.getServicePath(s), ctx.services)
```
Comment (mozart27) [inline Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py]: Is this lambda to get `tsdbs` just informational?  It appears it's overwritten in the next few lines.

Comment (ecifreo) [inline Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py]: It does get overwritten, but the first time called it gets all configs plus all "original configs", so yes, informational. Next time only gets "original configs".

File: Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py (hunk 1 above)
Comment (ecifreo) [inline Products/ZenModel/migrate/updateOpenTSDBSkipUnresolvedTags.py]: Copy pasta from another migration but I will fix, don't like typos.

**Pull request merged**